Sources/Sentry/Public/SentryOptions.h

```diff
@@ -264,6 +264,16 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableFileIOTracing;
 
+/**
+ * Indicates whether tracing should be enabled.
+ * Enabling this will change `tracesSampleRate` to 1 if both
+ * `tracesSampleRate` and `tracesSampler` are nil.
+ *
+ * Changing either `tracesSampleRate` or `tracesSampler` to
+ * a value other then nil will enable this in case this was never changed before.
+ */
+@property (nonatomic) BOOL enableTracing;
```

Comment:
`m`: This should be `nullable` and the default should be `nil`.

Reply:
Please, take a look at [this](https://github.com/getsentry/sentry-cocoa/pull/2693#issuecomment-1431449636) and [this](https://github.com/getsentry/sentry-cocoa/pull/2693#issuecomment-1431471982).